src/dlls/mscoree/unixinterface.cpp

```diff
@@ -86,76 +87,47 @@ static LPCWSTR* StringArrayToUnicode(int argc, LPCSTR* argv)
     return argvW;
 }
 
-static void ExtractStartupFlagsAndConvertToUnicode(
+static void InitializeStartupFlags(STARTUP_FLAGS* startupFlagsRef)
+{
+    STARTUP_FLAGS startupFlags = static_cast<STARTUP_FLAGS>(
+            STARTUP_FLAGS::STARTUP_LOADER_OPTIMIZATION_SINGLE_DOMAIN |
+            STARTUP_FLAGS::STARTUP_SINGLE_APPDOMAIN);
+
+    if (Configuration::GetKnobBooleanValue(W("System.GC.Concurrent"), CLRConfig::UNSUPPORTED_gcConcurrent))
```

Comment:
I think we don't really need to set this flag anymore. For CoreCLR, we're going to default to concurrent GC enabled anyway, so the flag doesn't really do anything. To disable concurrent GC, the user will either set "System.GC.Concurrent" to false or set the COMPlus value to false.

Any thoughts?